### PR TITLE
Bug fix for hi-res images using in imageView

### PR DESCRIPTION
**Pull request by afil310**
`imageView.contentMode` should be changed to `.scaleAspectFit` in order to be able to use hi-res images or to customize the imageView height by adjusting multiplier value in
`imageView.heightAnchor.constraint(equalTo: pageStackView.heightAnchor, multiplier: 0.5)`

If we have `imageView.contentMode = .center` and use hi-res image, the stack renders completely broken.

onboardkit before bug fix 500 onboardkit after bug fix 500

### CODE REVIEW

**Comment by NikolaKirev** [comment]
I understand the thinking behind this, but the reason I chose `.center` is because this way there will not be any upscaling.
My idea is that you would have specific images designed and prepared for the onboarding pages. and not use bigger ones than needed.
If you use `.scaleAspectFit` and you use an image that is (for example) 100 points tall and your ImageView is 200 points tall, the image will become upscaled and pixelated.

To summarise, I think the image assets used in the onboarding should be sized appropriately and in cases where they are smaller (by design) we want them centered and not upscaled.

Let me know if this makes sense. I'm happy to work on a solution that fixes both problems if there is one.

cc @afil310

**Comment by doodzik** [comment]
I think exposing the ability to change that setting and documenting it in the Readme would be a good solution. That way you can explain your reasoning behind it and provide the ability for someone to disagree by changing the default behaviour :)

**Comment by NikolaKirev** [comment]
As @doodzik suggested, this is now exposed as a customizable property in `AppearanceConfiguration`

This PR can be closed. Thanks for contributing!
